core/src/main/kotlin/org/evomaster/core/output/service/naming/TestCaseNamingStrategy.kt

```diff
@@ -0,0 +1,16 @@
+package org.evomaster.core.output.service.naming
+
+import org.evomaster.core.output.NamingHelper
+import org.evomaster.core.output.TestCase
+import org.evomaster.core.search.EvaluatedIndividual
+import org.evomaster.core.search.Solution
+
+abstract class TestCaseNamingStrategy(
```

Comment:
for abstract class and interfaces, add JavaDocs for each method

Reply:
Added